spec/index.html

```diff
@@ -729,8 +730,9 @@ <h2>Literals</h2>
 
     <p>A literal is a <dfn>language-tagged string</dfn> if the third element
       is present and the fourth element is not present.
-      Lexical representations of language tags MAY be converted
-      to lower case.
+      Lexical representations of language tags
+      MUST be case normalized
+      and MAY (preferably) be converted to lower case.
       The value of language tags is always treated as being in lower case.</p>
```

Comment:
```suggestion
      </p>
```
Sentence no longer needed.